src/Microsoft.Extensions.Configuration.AzureAppConfiguration/AzureAppConfigurationProvider.cs

```diff
@@ -228,7 +229,7 @@ public async Task RefreshAsync(CancellationToken cancellationToken)
                             foreach (KeyValueWatcher changeWatcher in cacheExpiredWatchers)
                             {
                                 string watchedKey = changeWatcher.Key;
-                                string watchedLabel = changeWatcher.Label;
+                                string watchedLabel = changeWatcher.Label.NormalizeNull();

```

Comment:
Why is `NormalizeNull` needed? I believe it already takes place inside of KeyValueIdentifier.

Reply:
I thought the reason might be because we pass watchedLabel into GetConfigurationSettingAsync, if not then I can remove it

Reply:
Well the main branch today doesn't NormalizeNull here, so is this a bug you're fixing?

Reply:
It wasn't a specific bug, I was just told to call normalizenull on anywhere we used changewatcher.label in the map pr

Reply:
Right now it looks orthogonal to the watched settings update and also seems like it would have some behavioral change.

Reply:
My bad, I missed that we are calling it inside KeyValueIdentifier. It can be removed from both places in this PR.